css/Components/Hero.scss

```diff
@@ -0,0 +1,131 @@
+@import "../Theme/Color";
+@import "../Theme/Spacing";
+@import "../Theme/Breakpoint";
+
+$Hero-max-width: 1211px;
+$Hero-uvp-max-width: 523px;
+$Hero-z-index-background: 0;
+$Hero-z-index-foreground: 1;
+
+.Hero {
+  position: relative;
+
+  @include Breakpoint-desktopOnly {
+    padding: 0 $Spacing-small;
+  }
+}
+
+.Hero-background {
+  position: absolute;
+
+  top: 0;
+  left: 0;
+
+  width: 100%;
+  height: 100;
+
+  z-index: $Hero-z-index-background;
+}
+
+.Hero-foreground {
+  padding-top: $Spacing-base;
+  padding-bottom: $Spacing-xxSmall;
+
+  z-index: $Hero-z-index-foreground;
+
+  @include Breakpoint-desktopOnly {
+    max-width: $Hero-max-width;
+
+    padding-top: $Spacing-large;
+
+    margin: 0 auto;
+  };
+}
+
+.Hero-content {
+  padding: 0 $Spacing-xSmall;
+
+  @include Breakpoint-desktopOnly {
+    display: flex;
+    align-items: center;
+
+    padding: 0 $Spacing-large;
+  };
+}
+
+.Hero-uvp {
+  margin-top: $Spacing-xSmall;
+  margin-bottom: $Spacing-xxSmall;
+
+  @include Breakpoint-desktopOnly {
+    max-width: $Hero-uvp-max-width;
+  }
+}
+
+.Hero-title {
+  @include Breakpoint-desktopOnly {
+    flex: 2;
+
+    padding-left: $Spacing-small
+  };
+}
+
+.Hero-actions {
+  margin-top: $Spacing-base;
+
+  @include Breakpoint-desktopOnly {
+    flex: 1;
+
+    margin: 0;
+
+    text-align: right;
+  };
+}
+
+.Hero-playButton {
+  position: relative;
+
+  padding-left: $Spacing-xxSmall;
+
+  cursor: pointer;
+
+  @include Breakpoint-desktopOnly {
+    display: inline-block;
+
+    text-align: left;
+  }
+}
+
+.Hero-playButtonImage {
+  position: absolute;
+
+  top: 0;
+  left: 0;
+
+  height: 100%;
+}
+
+.Hero-playButtonLabel {
+  position: relative;
```

Comment:
I'm looking at this and the browser inspector, and can't understand the reason for these relative positionings. can you explain?